src/runtime/virtcontainers/persist/api/network.go

```diff
@@ -90,6 +94,7 @@ type NetworkEndpoint struct {
 	Tap       *TapEndpoint       `json:",omitempty"`
 	IPVlan    *IPVlanEndpoint    `json:",omitempty"`
 	Tuntap    *TuntapEndpoint    `json:",omitempty"`
+	Vfio      *VfioEndpoint      `json:",omitempty"`
```

Comment:
@l8huang We currently have `PhysicalEndpoint` to support and detect SRIOV network interface or an entire physical network interface. 
Is that different from your use-case?

Reply:
PhysicalEndpoint is for VF device, which presents a network interface on host; this PR is for VFIO passthrough device, on host it doesn't has a network interface, it supposes to be plugged into guest VM and recognized by NIC driver in guest kernel to create its network interface.